scripts/init_model.py

```diff
@@ -0,0 +1,174 @@
+# coding: utf8
+
+import os
+import sys
+import plac
+import math
+from tqdm import tqdm
+import numpy
+from ast import literal_eval
+from pathlib import Path
+from preshed.counter import PreshCounter
+import tarfile
+import gzip
+import zipfile
+import json
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir))))
+
+from spacy.cli._messages import Messages
+from spacy.vectors import Vectors
+from spacy.errors import Errors, Warnings, user_warning
+from spacy.util import prints, ensure_path, get_lang_class
+
+from scispacy.file_cache import cached_path
+
+@plac.annotations(
+        lang=("model language", "positional", None, str),
+        output_dir=("model output directory", "positional", None, Path),
+        freqs_loc=("location of words frequencies file", "positional", None, Path),
+        vectors_loc=("optional: location of vectors file in Word2Vec format "
+                     "(either as .txt or zipped as .zip or .tar.gz)", "option",
+                     "v", str),
+        no_expand_vectors=("optional: Whether to expand vocab with words found in vector file",
+                           "flag", "x", bool),
+        meta_overrides=("optional: meta_json file to load.",
+                        "option", "m", Path),
+        prune_vectors=("optional: number of vectors to prune to",
+                       "option", "V", int)
+)
+def init_model(lang, output_dir, freqs_loc=None,
+               vectors_loc=None, no_expand_vectors=False,
+               meta_overrides=None, prune_vectors=-1):
+    """
+    Create a new model from raw data, like word frequencies, Brown clusters
+    and word vectors.
+    """
+    if vectors_loc is not None:
+        vectors_loc = cached_path(vectors_loc)
+    freqs_loc = cached_path(freqs_loc)
+
+    if freqs_loc is not None and not freqs_loc.exists():
+        prints(freqs_loc, title=Messages.M037, exits=1)
+    vectors_loc = ensure_path(vectors_loc)
```

Comment:
does this line also need a none check? (I didn't go look at the spacy code for this utility, so maybe its fine)